include/constants/common_strings_hints.h

```diff
@@ -0,0 +1,13 @@
+#ifndef GUARD_COMMON_STRINGS_HINTS_H
+#define GUARD_COMMON_STRINGS_HINTS_H
+
+// size: 0x8
+typedef struct Hints
```

Comment:
Not really a fan of having structs in `constants`. Defeats the purpose of the directory name.

Reply:
Honestly I didn't know pokeemerald put them in global.h. I was just used to doing it this way and saw pokediamond did it that way. I'm not sure what to do now I guess, open to suggestions. But the structs do need to be separate from the c/h code files, that's for certain.
As for the common_strings ones, I also wasn't sure where else to put the hints one since I was already following the pattern from the previous ones in common_strings. Also open to suggestions for it, or we can leave them for my next PR. The entire structs/includes thing still needs the most work out of everything we're doing

Reply:
> Also open to suggestions for it, or we can leave them for my next PR.

We can just leave it for now.